ds_judgements_public_ui/templates/layouts/base.html

```diff
@@ -108,6 +108,22 @@ <h1 class="service-introduction__header">Find case law<span class="phase-banner_
         </header>
       {% endflag %}
 
+      {% flag v3_homepage %}
```

Comment:
This makes it so I see the header twice on all pages apart from the home page:

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/2bcaec9c-3618-4cca-8166-e09c623e7f45">

Reply:
@jlhdxw I have sent you the urls to view each different version of homepage on slack